File: app/src/main/java/com/example/requestapp/ui/fragment/addTaskFragment/AddTaskFragmentContract.java
package com.example.requestapp.ui.fragment.addTaskFragment;

import com.example.requestapp.model.Task;

public interface AddTaskFragmentContract {

    interface Presenter{

        void onAddTaskClicked(Task task);

    }
    interface View{

        void showMessage(String message);

    }

}

File: app/src/main/java/com/example/requestapp/ui/fragment/addTaskFragment/AddTaskFragmentContract.kt
package com.example.requestapp.ui.fragment.addTaskFragment

import com.example.requestapp.model.Task

interface AddTaskFragmentContract {
    interface Presenter {
        fun onAddTaskClicked(task: Task?)
    }

    interface View {
        fun onAddedSuccessfulToast(message: String?)
        fun onAddedFailureToast(message: String?)
    }
}